tests/evals/runner/test_resume.py

```
@@ -0,0 +1,326 @@
+        load_resume_skip_keys(p, label="local", model="new-standard-id")
+
+
+def test_load_resume_skip_keys_truncated_json(tmp_path: Path, capsys):
+    p = tmp_path / "out.jsonl"
+    p.write_text(
+        json.dumps({"task_id": "R1", "rep": 0, "label": "local", "error": None})
+        + "\n"
+        + '{"task_id": "W1", "rep": 0, "label": "local", "error":\n',  # truncated
+        encoding="utf-8",
+    )
+    skip, n_skip, n_retry = load_resume_skip_keys(p, label="local")
```

Ruff (0.16.1)

[warning] Unpacked variable `n_retry` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)